user_templates/mcu_defaults/ercf/BTT_MMB_CAN_v1.0.cfg

```diff
@@ -0,0 +1,31 @@
+
+#---------------------------------------------#
+#### BTT MMB CAN 1.0 MCU definition ###########
+#---------------------------------------------#
+
+[mcu ercf]
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+# canbus_uuid: change-me-to-the-correct-canbus-id
+##--------------------------------------------------------------------
+
+# If you want to override the wiring of the Fystec ERCF ERB, keep in mind that this
```

Comment:
Please change the name of the board here to the correct name (probably a copy/paste typo)